src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/RoshSummaryTransformer.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.OASysAssessmentState
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.OASysRiskOfSeriousHarmSummary
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.oasyscontext.RoshSummary
+
+@Component
+class RoshSummaryTransformer : OASysTransformer() {
+  fun transformToApi(roshSummary: RoshSummary) = OASysRiskOfSeriousHarmSummary(
+    assessmentId = roshSummary.assessmentId,
+    assessmentState = if (roshSummary.dateCompleted != null) OASysAssessmentState.completed else OASysAssessmentState.incomplete,
+    dateStarted = roshSummary.initiationDate,
+    dateCompleted = roshSummary.dateCompleted,
+    riskOfSeriousHarmSummary = listOf(
+      oASysQuestionWithSingleAnswer("Who at Risk", "R10.1", roshSummary.roshSummary.whoAtRisk),
+      oASysQuestionWithSingleAnswer("Nature of Risk", "R10.2", roshSummary.roshSummary.natureOfRisk),
+      oASysQuestionWithSingleAnswer("Risk Greatest", "R10.3", roshSummary.roshSummary.riskGreatest),
+      oASysQuestionWithSingleAnswer("Risk Increase Likely To", "R10.4", roshSummary.roshSummary.riskIncreaseLikelyTo),
+      oASysQuestionWithSingleAnswer("Risk Reduction Likely To", "R10.5", roshSummary.roshSummary.riskReductionLikelyTo)
```

Comment:
It might be slightly more helpful to use the actual form of words from OASys? They are:

"Who is at risk"
"What is the nature of the risk"
"When is the risk likely to be the greatest"
"What circumstances are likely to increase risk"
"What circumstances are likely to reduce the risk"

Reply:
Ah these make a lot more sense 👍